web/html/constants.js

```diff
@@ -0,0 +1,2 @@
+// auto generated file by gen.go 
```

Comment:
Should it be commited if it's go generated?

Reply:
Why do you need this file ? Why don't you:
- define an endpoint to return these constant at the helloworld level
- bind a new endpoint in this project using the handler of hello world ?

Reply:
yes it must be commited to follow good practices.

I have generated this file from golang code so that the constants are only defined at one place: golang code, and accessible directly in the javascript code. (btw I know that the front end will be entirely rework)

I don't understand you proposal: because the constants are deffirent from one plugin to another, why the wallet would use a handler of the hello-world? Also, my front end app need these constants to perform any http call to thyra so I can't get these variable through API call.